heareval/score.py

```diff
@@ -223,6 +225,83 @@ class EventBasedScore(SoundEventScore):
     score_class = sed_eval.sound_event.EventBasedMetrics
 
 
+class MeanAveragePrecision(ScoreFunction):
```

Comment:
If you're going to use someone else's code, you should reference them and also indicate where the code is from

Reply:
I think this is resolved now?